apps/core/test/services/repositories_test.exs

```diff
@@ -57,9 +70,18 @@ defmodule Core.Services.RepositoriesTest do
       %{owner: user} = publisher = insert(:publisher)
       repo = insert(:repository, publisher: publisher)
 
-      {:ok, updated} = Repositories.update_repository(%{name: "piazza"}, repo.id, user)
+      {:ok, updated} = Repositories.update_repository(%{
+        name: "piazza",
+        community: %{
+          twitter: "twitter.com/piazza",
+          videos: ["https://www.youtube.com/watch?v=dQw4w9WgXcQ"]
+        }
+      }, repo.id, user)
 
       assert updated.name == "piazza"
+      assert updated.community.twitter == "twitter.com/piazza"
+      assert length(updated.community.videos) == 1
```

Comment:
you should be able to just do `asset updated.community.videos == ["https://www.youtube.com/watch?v=dQw4w9WgXcQ"]` I believe

Reply:
True, makes no sense to split it into 2 lines. Fixed.